src/wp-includes/script-loader.php

```diff
@@ -1843,6 +1843,38 @@ function wp_just_in_time_script_localization() {
 	);
 }
 
+/**
+ * Prints a loader script if there is text/template registered script.
+ *
+ * @since 6.3.0
+ *
+ * When added to the DOM, this script converts any text/template scripts
+ * associated with a handle into type/javascript, and executes them.
+ */
+function wp_print_delayed_inline_script_loader() {
+	$wp_scripts = wp_scripts();
+
+	if ( $wp_scripts->has_delayed_inline_script() ) {
+		$output    = <<<JS
+function wpLoadAfterScripts(handle) {
```

Comment:
@westonruter @felixarntz @joemcgill I've updated this function body (and accompanying tests) based on @westonruter 's suggestion. I tested locally and all continued to function as per the prior implementation.